components/automate-deployment/pkg/server/automate_config.go

```diff
@@ -409,3 +451,14 @@ func configLogrotate(req *config.Log) error {
 func rollbackLogrotate() error {
 	return os.Remove(logRotateConfigFile)
 }
+
+func UpdateOfLogroateConfig(req *api.PatchAutomateConfigRequest, existingCopy *deployment.AutomateConfig) (*api.PatchAutomateConfigRequest, error) {
```

Comment:
Can we rename this function to something more apt? Like comparingtheconfigs? are we actually updating the configs here?